spec/lib/common/db-renderable-content-spec.js

```diff
@@ -0,0 +1,254 @@
+            return self.subject.put('test thing', 'test contents')
+            .then(function() {
+                expect(waterline.things.create).to.have.been.calledWith({
+                    name: 'test thing',
+                    contents: 'test contents',
+                    scope: 'global'
+                });
+            });
+        });
+
+        it('should create a new thing in the specified scope', function() {
+            var self = this;
+            waterline.things.findOne.resolves(null);
+            return self.subject.put('test thing', 'test contents', 'sku')
+            .then(function() {
+                expect(waterline.things.create).to.have.been.calledWith({
+                    name: 'test thing',
+                    contents: 'test contents',
+                    scope: 'sku'
+                });
+            });
+        });
+
+        it('should update an existing thing', function() {
+            var self = this;
+            waterline.things.findOne.resolves('not empty');
+            return self.subject.put('test thing', 'test contents', 'sku')
+            .then(function() {
+                expect(waterline.things.update).to.have.been.calledWith(
+                    { name: 'test thing', scope: 'sku' },
+                    { contents: 'test contents' }
+                );
+            });
+        });
+
+        it('should delete a thing in the global scope', function() {
+            var self = this;
+            waterline.things.destroy.resolves();
+            return self.subject.unlink('it')
+            .then(function() {
+                expect(waterline.things.destroy).to.have.been.calledWith(
+                    { name: 'it', scope: 'global' }
+                );
+            });
+        });
+
+        it('should delete a thing in the specified scope', function() {
+            var self = this;
+            waterline.things.destroy.resolves();
+            return self.subject.unlink('it', 'scope')
+            .then(function() {
+                expect(waterline.things.destroy).to.have.been.calledWith(
+                    { name: 'it', scope: 'scope' }
+                );
+            });
+        });
+    });
+
+    describe('failure logging', function() {
+        var nodeEnv;
+
+        before(function() {
+            nodeEnv = process.env.NODE_ENV;
+        });
+
+        beforeEach(function() {
+            loader.prototype.getAll.rejects(new Error());
+        });
+
+        after(function() {
+            process.env.NODE_ENV = nodeEnv;
+        });
+
+        it('should log warnings on load failure', function() {
+            process.env.NODE_ENV = 'not test';
+            return this.subject.load()
+            .then(function() {
+                expect(Logger.prototype.log).to.have.been.calledWithMatch(
+                    'error', /Unable to load things/, { error: new Error() });
+            });
+        });
+
+        it('should not log warnings in test env', function() {
```

Comment:
Heh, legacy is all about John's setting up for the globing/loading in the on-core libraries if I remember. I think he set up this pattern to not have to see the log messages in running tests (which where a combo of unit and functional) where they "weren't useful", as at the time we have logging extremely tightly tied to the messaging layer rather than as a layer where it could shim in as needed. 

My own opinion is that we could easily change this - as long as the warning message really is a warning and not set at the wrong level, I personally kinda like to see them, even in log output from unit tests